charts/osm/values.yaml

```diff
@@ -470,7 +470,7 @@ osm:
     enableWASMStats: true
     # -- Enable OSM's Egress policy API.
     # When enabled, fine grained control over Egress (external) traffic is enforced
-    # DEPRECATED. To use EgressPolicy, disable global mesh-wide egress using '--set osm.enableEgress=false'.
+    # DEPRECATED, do not use. To use EgressPolicy API, disable global mesh-wide egress using '--set osm.enableEgress=false'.
```

Comment:
do we need to maintain cli flags as part of our deprecation policy?

Reply:
This is a Helm chart value that can be set using an opaque `--set` CLI flag. We need to keep this property at least for 1 more release after announcing deprecation, after which we can drop it completely. It's the safer thing to do.